spec/requests/users_controller_spec.rb

```diff
@@ -1297,6 +1297,8 @@ def post_user(extra_params = {})
       before do
         UsersController.any_instance.stubs(:honeypot_value).returns("abc")
         UsersController.any_instance.stubs(:challenge_value).returns("efg")
```

Comment:
Do we still need the stubs here?

Reply:
yes, SessionController is setting honeypot_value and challenge_value, and UsersController is checking that values are correct